.github/pull_request_template.md

```diff
@@ -12,8 +12,6 @@
 
 ### Fixed
 
--
-
----
+- ***
```

Comment:
I'm a bit confused, can you explain what this change is for? The way I see it you removed the horizontal line between the change log and the extra notes (ticket etc), and added `***` in the first bullet point of `Fixed`?

Reply:
Hmm, this was done by prettier. Probably because it's an empty list

Reply:
I'll revert it